src/aof.c

```diff
@@ -799,7 +806,11 @@ int loadAppendOnlyFile(char *filename) {
 
     /* This point can only be reached when EOF is reached without errors.
      * If the client is in the middle of a MULTI/EXEC, log error and quit. */
```

Comment:
Note that this was the old behavior, and the comment still reflects the old one... But later we changed it.